managers/graph-operations-manager.js

```diff
@@ -8,6 +16,119 @@ class GraphOperationsManager {
+     * @param {boolean} [options.validate] - Whether to validate the retrieved assertion.
+     * @param {string} [options.outputFormat] - The format of the retrieved assertion output, either "n-quads" or "json-ld".
+     * @returns {Object} - The result of the asset get operation.
+     */
+    async get(UAL, options = {}) {
+        const {
+            blockchain,
+            endpoint,
+            port,
+            maxNumberOfRetries,
+            frequency,
+            state,
+            contentType,
+            validate,
+            outputFormat,
+            authToken,
+            hashFunctionId,
+            paranetUAL,
+        } = this.inputService.getAssetGetArguments(options);
+
+        this.validationService.validateAssetGet(
+            UAL,
+            blockchain,
+            endpoint,
+            port,
+            maxNumberOfRetries,
+            frequency,
+            state,
+            contentType,
+            hashFunctionId,
+            validate,
+            outputFormat,
+            authToken,
+        );
+
+        const getOperationId = await this.nodeApiService.get(
+            endpoint,
+            port,
+            authToken,
+            UAL,
+            state,
+            contentType,
+            hashFunctionId,
+            paranetUAL,
+        );
+
+        const getOperationResult = await this.nodeApiService.getOperationResult(
+            endpoint,
+            port,
+            authToken,
+            OPERATIONS.GET,
+            maxNumberOfRetries,
+            frequency,
+            getOperationId,
+        );
+
+        const assertion = getOperationResult.data.assertion;
+
+        if (!assertion) {
+            if (getOperationResult.status !== 'FAILED') {
+                getOperationResult.data = {
+                    errorType: 'DKG_CLIENT_ERROR',
+                    errorMessage: 'Unable to find assertion on the network!',
+                };
+                getOperationResult.status = 'FAILED';
+            }
+
+            return {
+                operation: {
+                    get: getOperationStatusObject(
+                        getOperationResult,
+                        getOperationId,
+                    ),
+                },
+            };
+        }
+
+        if (validate === true) {
+            const isValid = true; // TODO: validate assertion
+            if (!isValid) {
+                getOperationResult.data = {
+                    errorType: 'DKG_CLIENT_ERROR',
+                    errorMessage: "Calculated root hashes don't match!",
+                };
+            }
+        }
+
+        let formattedAssertion;
+        if (outputFormat === GET_OUTPUT_FORMATS.JSON_LD) {
+            formattedAssertion = await toJSONLD(assertion.join('\n'));
+        }
+        if (outputFormat === GET_OUTPUT_FORMATS.N_QUADS) {
+            formattedAssertion = await toNQuads(assertion, 'application/n-quads');
```

Comment:
it might be already in nquads, you can debug this i guess (?) maybe im wrong also

Reply:
yeah I was confused because of this https://github.com/OriginTrail/dkg.js/blob/v8/develop/managers/asset-operations-manager.js#L722 @u-hubar can you maybe explain this line